docs/stencil-docs/customizing-checkout/checkout-sdk-example.mdx

```diff
@@ -1,8 +1,8 @@
 # Checkout SDK Tutorial
 
-This tutorial assumes you are familiar with the Checkout SDK. If you are unfamiliar with the Checkout SDK, refer to [Getting Started with Checkout SDK](/docs/storefront/cart-checkout/checkout-sdk/tutorial) to get some general information. 
+This tutorial assumes you are familiar with the Checkout SDK. If you are unfamiliar with the Checkout SDK, refer to [Getting Started with Checkout SDK](/docs/stencil-docs/customizing-checkout/checkout-sdk.mdx) to get some general information.
```

Comment:
I tested the links and /docs/stencil-docs/customizing-checkout/checkout-sdk.mdx doesn't work.

Reply:
```suggestion
This tutorial assumes you are familiar with the Checkout SDK. If you are unfamiliar with the Checkout SDK, refer to [Getting Started with Checkout SDK](/docs/storefront/cart-checkout/checkout-sdk) to get some general information.
```